src/client/client.tsx

```diff
@@ -0,0 +1,22 @@
+import UserJSON from "../data/users.json";
+import EventsJSON from "../events.json";
+import { Event, Member } from "../util/Types";
+
+export function fetchEvent(id: Number): Promise<Event> {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      const event = (EventsJSON as unknown as Event[]).find((e) => e.id === id);
+      event ? resolve(event) : reject("404 Not found");
+    }, 1000);
+  });
+}
+
+export function fetchMember(id: String): Promise<Member> {
+  console.log("PLEASEEEEE");
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      const member = (UserJSON as unknown as Member[]).find((m) => m.id === id);
```

Comment:
do you need to search here by NUID rather than the ID? since the string passed in is the NUID, but you're searching here on the ID, it will never find the right nuid?